LocationServices/LocationServices.xcodeproj/LocationServices.xctestplan

```diff
@@ -18,13 +18,22 @@
   },
   "testTargets" : [
     {
-      "target" : {
-        "containerPath" : "container:LocationServices.xcodeproj",
-        "identifier" : "963EBF4E29AF92F5001961F2",
-        "name" : "LocationServicesTests"
-      }
-    },
-    {
+      "skippedTests" : [
```

Comment:
Why are those tests skipped?

Reply:
They are redundant tests. For example AWS Connection & Sign in is already covering in Tracking & Geofence tests. Same goes for Add & Delete geofence which is covered in Edit geofence. If we execute all tests individually it would take somewhere between 90-180 minutes. Right now they are passing within 50-60 mins